changes/3296.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed a bug that caused apps on linux to crash with Gtk3
```

Comment:
Support for GTK3 installs that use a GIO release older than 2.72 has been restored. Ubuntu 22.04, and other Debian 12-derived systems are affected by this issue.